tox.ini

```diff
@@ -3,26 +3,32 @@ envlist = py26,py27,py33,py34,pypy,flake8
 
 [testenv]
 deps= -rdev_requirements.txt
+# To make sure we actually test the tarball and now the working directory
+changedir = .tox
 commands=haas okonomiyaki
 
 [testenv:pypy]
 deps= -rdev_requirements2.txt
+changedir = .tox
 
 [testenv:py27]
 deps= -rdev_requirements2.txt
+changedir = .tox
 
 [testenv:py26]
 deps= -rdev_requirements2.txt
+changedir = .tox
 
 [testenv:coverage]
 deps= -rdev_requirements2.txt
-commands=coverage erase
-         coverage run -m haas
-         coverage report --include=okonomiyaki/*
+changedir = .tox
+commands=coverage erase --rcfile=../.coveragerc
+         coverage run --rcfile=../.coveragerc -m haas
+         coverage report --rcfile=../.coveragerc --include=okonomiyaki/*
 
 [testenv:flake8]
 deps= -rdev_requirements2.txt
-commands=flake8 okonomiyaki
+commands=flake8 ../okonomiyaki
```

Comment:
No, actually it was not needed in any of the `testenv:` sections.

I removed it + fixed a coverage issue